scss/chat.scss

```diff
@@ -56,3 +72,9 @@ body.has-full-page-chat {
 .chat-drawer-active.chat-drawer-expanded .chat-composer-dropdown__menu-content {
   z-index: z("composer", "content") + 1;
 }
+
+.chat-replying-indicator-container {
+  @include breakpoint("mobile-extra-large") {
```

Comment:
We should update this to the new viewport syntax. `@include breakpoint` is already removed from core CSS, and will soon be linted against in themes/plugins too.